health/service/src/test/java/com/cerner/beadledom/health/HealthServiceModule.java

```diff
@@ -0,0 +1,36 @@
+package com.cerner.beadledom.health;
+
+import com.cerner.beadledom.metadata.BuildInfo;
+import com.cerner.beadledom.metadata.ServiceMetadata;
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import java.time.Instant;
+import java.util.Properties;
+
+/**
+ * A Module that installs the health module and provides the service metadata
+ *
+ * @author Nick Behrens
+ */
+public class HealthServiceModule extends AbstractModule {
+  @Override
+  public void configure() {
+    install(new HealthModule());
+  }
+
+  @Provides
+  ServiceMetadata provideServiceMetadata() {
+    return ServiceMetadata.builder()
+        .setBuildInfo(
+            BuildInfo.builder()
+                .setArtifactId("Lombre")
```

Comment:
Lets use more realistic values

Reply:
Changed the values here: 82d60f08015ffdfdef327b1e929f9dc901c880b3